mopidy_local_images/library.py

```diff
@@ -228,3 +219,6 @@ def _scan(self, uri):
         logger.debug('Scanning %s for images', uri)
         data = self.scanner.scan(uri)
         return data.get('tags', {})
+
+    def get_data_dir(self, config):
+        raise NotImplementedError  # overridden by Extension
```

Comment:
Can you explain a bit more... You mean split the get_or_migrate_data_dir() into two functions?